### PR TITLE
Budget App Features

### LINKED ISSUE #2: Create ERD Diagram
Create your models according to the ERD diagram. Don't forget about validations and tests.

### LINKED ISSUE #3: Set up the repository and tools.
- Linter
- GitHub Flows

### LINKED ISSUE #4: Implement the authentication and authorization
Add devise and page

### LINKED ISSUE #5: Create the page to add a category.

### LINKED ISSUE #6: Create the home page.

### LINKED ISSUE #7: Create the page to add a transaction

### LINKED ISSUE #8: Create the transactions page

### LINKED ISSUE #9: Make sure your app is tested adequately

### LINKED ISSUE #10: Deploy the project, and test for final details

### LINKED ISSUE #11: Record a video for your project

### PR DESCRIPTION
**Pull request by bushmusi**
> # Below features have been added

1. Splash screen
2. Sign up and log in to pages
3. Home page (categories page)
4. Transactions page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Budget-App is about building a mobile web application where you can manage your
 # Live Demo
   - [budget-app](https://bush-budget.herokuapp.com/)
```
Comment (elyor-doniyorov) [inline README.md:11]: - [ ] Kindly record a a video presenting the features of the project. Also make sure that :point_down: 
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @bushmusi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: The README contains an "Acknowledgements" section
:heavy_check_mark: The project is deployed properly
:heavy_check_mark: etc.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bushmusi** [comment]
Hi @elyor-doniyorov , Thanks for reminding here is the recorded video [link ](https://www.loom.com/share/01826d1d18be4c69bfa67a63671e3150)

**Review by elyor-doniyorov — APPROVED** [review]
Hi @bushmusi again, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**